spec/unit/database/database.spec.js

```diff
@@ -0,0 +1,20 @@
+var db_config = require("./config");
+describe("Database", function() {
+    describe("Application", function() {
+    	beforeEach(function(){
+			var node_env = process.env.NODE_ENV;
```

Comment:
Identifier 'node_env' is not in camel case.<br>'process' is not defined.<br>'node_env' is defined but never used.